osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -139,6 +141,26 @@ private void load()
             updateCatcher();
         }
 
+        /// <summary>
+        /// Calculates the scale of the catcher based off the provided beatmap difficulty.
+        /// </summary>
+        internal static Vector2 CalculateScale(BeatmapDifficulty difficulty)
```

Comment:
private?

Reply:
it's a method for calculating same as the others so wasn't sure about making it private, will do.

Reply:
if a method is not used externally (and doesn't carry any foreseeable purpose) then it should be private.